.github/workflows/db_rollback.yml

```diff
@@ -0,0 +1,68 @@
+name: Rollback DB
+
+on:
+  workflow_dispatch:
+    inputs:
+      rollbackCount:
+        description: 'Number of migrations to rollback'
+        required: true
+        default: # of rollback you require in the file input
+      rollbackFile:
+        description: "File with migrations you want to rollback"
+        required: true
+        default: Filepath to yml
+
+jobs:
+  terraform-deploy:
+    name: Terraform Deploy
+    environment:
+      name: ${{ inputs.ENVIRONMENT }}
+    runs-on: ubuntu-latest
+    env:
+      ARM_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
+      ARM_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+      ARM_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      ARM_USE_OIDC: true
+    permissions:
+      id-token: write
+      contents: read
+    defaults:
+      run:
+        working-directory: ${{ inputs.TERRAFORM_DIRECTORY }}
```

Comment:
Another `inputs.` that won't work in this context.  This might be something like `./operations/environments/${{ github.event.inputs.environment }}`?